src/ImageSharp.Web/Middleware/ImageContext.cs

```diff
@@ -125,12 +125,11 @@ public void ComprehendRequestHeaders(DateTimeOffset lastModified, long length)
         /// <param name="statusCode">The status code.</param>
         /// <param name="metaData">The image metadata.</param>
         /// <returns>The <see cref="Task"/>.</returns>
-        public Task SendStatusAsync(int statusCode, in ImageCacheMetadata metaData)
```

Comment:
`async` methods can't have `in` parameters, so I've removed `in` from the signature

Reply:
The method doesn't actually have to be async, we can pass the Task along directly.